cypress/e2e/visual-regression.cy.ts

```diff
@@ -0,0 +1,40 @@
+// This test has a few oddities
+// - Enable automatic snapshots at the end of each test by disabling the disable.
+// - Scroll to bottom in each test to ensure all lazy loading images are ready.
+//   afterEach() is too late.
+describe("Site visuals", { env: { disableAutoSnapshot: false } }, () => {
+  it("renders the frontpage", () => {
+    cy.visit("/");
+    cy.scrollTo("bottom", { duration: 2000 });
+  });
+
+  it("renders the article list", () => {
+    cy.visit("/articles");
+    cy.scrollTo("bottom", { duration: 2000 });
+  });
+
+  it("renders an article page", () => {
+    cy.visit("/by_uuid/node/2cd0fe5e-4159-4452-86aa-e1a1ac8db4a1");
```

Comment:
I would avoid using these IDs - I think they might change at some point, when we update the example content.

You could also go to the /articles page and search for the title of the content, and then click on it from there?

Reply:
I think it is appropriate to use these ideas. They provide a direct link between the tests and the example content.

The UUID for an entity stays the same even if it is updated. I think that is a good thing compared to navigating through other pages.

We use the same approach for [Lighthouse](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/develop/.lighthouserc.js) and [Pa11y](https://github.com/danskernesdigitalebibliotek/dpl-cms/blob/develop/.pa11yci) tests. Using it here for direct entity access is consistent.